### PR TITLE
Shuffling letters before assigning them to training and validation datasets

### PR DESCRIPTION
**Pull request by lukas-krecan**

### CODE REVIEW

**Comment by lukas-krecan** [comment]
I signed it!

**Comment by vincentvanhoucke** [comment]
Sounds like a good idea. Can you:
- replace "lets" with "let's"
- move the call to np.random.seed(133) to the top of the file, so that the seed is fixed for everything
- add a:
valid_dataset, valid_labels = randomize(valid_dataset, valid_labels) where the other calls are

I think that would take care of all randomization and reproducibility issues.

**Comment by lukas-krecan** [comment]
Sure, I just do not get what do you mean by the third bullet point "add a: valid_dataset, valid_labels = randomize(valid_dataset, valid_labels) where the other calls are"

**Comment by vincentvanhoucke** [comment]
We don't appear to be re-randomizing the validation set here:
train_dataset, train_labels = randomize(train_dataset, train_labels)
test_dataset, test_labels = randomize(test_dataset, test_labels)

**Comment by lukas-krecan** [comment]
I see. Should be OK now

**Comment by vincentvanhoucke** [comment]
Merged. Thanks!
